packages/api/test/server-test.js

```diff
@@ -12,18 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-process.env.SESSION_SECRET = 'fake secret';
 process.env.FRONTEND_URL = 'https://flaky-dashboard.web.app/home';
-const { describe, it } = require('mocha');
+const { describe, it, afterEach } = require('mocha');
 
 const assert = require('assert');
 const fetch = require('node-fetch');
+const sinon = require('sinon');
+const querystring = require('querystring');
+const auth = require('../src/auth.js');
+const repo = require('../src/repository.js');
+// const nock = require('nock');
+
+// nock('http://github.com')
```

Comment:
likewise, I think we shouldn't need to use nock for the new tests that are added, let's just get rid of the commented out code.